### PR TITLE
Add metrics for metric repopulation

### PR DESCRIPTION
**Pull request by rudigiesler**
Since we added the framework for repopulating metrics here: https://github.com/praekelt/hellomama-registration/pull/100 , we now need to add the actual calculation of the individual metrics.

### CODE REVIEW

**Comment by rudigiesler** [comment]
@KaitCrawford Ready for review.

File: registrations/test_metrics.py [hunk 1]
```diff
@@ -23,6 +30,474 @@ def test_generate_metric(self):
...
+        user1 = User.objects.create(username='user1')
+        user2 = User.objects.create(username='user2')
+        source1 = Source.objects.create(
+            name='TestSource1', authority='hw_full', user=user1)
+        source2 = Source.objects.create(
+            name='TestSource2', authority='hw_full', user=user2)
+
+        url = 'http://localhost:8001/api/v1/identities/search/?' \
+              'details__state=state1'
+        responses.add_callback(
```
Comment (KaitCrawford) [inline registrations/test_metrics.py]: This one doesn't contact the identity store does it?

Comment (rudigiesler) [reply registrations/test_metrics.py]: Nope, the test doesn't even have the responses decorator so this does nothing. I'll remove it.

Comment (rudigiesler) [reply registrations/test_metrics.py]: Resolved with https://github.com/praekelt/hellomama-registration/pull/101/commits/e7f1692586f598cb09cd00d0a54f054badbec432

**Comment by rudigiesler** [comment]
@KaitCrawford Ready for another review.

**Comment by KaitCrawford** [comment]
:+1:  :cake: :cake: :cake: awesome work!

**Pull request merged**